eng/versioning/version_client.txt

```diff
@@ -353,6 +353,9 @@ com.azure.tools:azure-sdk-build-tool;1.0.0-beta.1;1.0.0-beta.2
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+unreleased_com.azure:azure-core;1.29.0-beta.1
```

Comment:
There should be no changes to core. The changes should only be in core-amqp.

Reply:
Sorry, the previous change is in core, I forgot to remove this.